translations/messages.nl.xlf

```diff
@@ -1776,5 +1776,23 @@
         <target>Weet je zeker dat je deze content wilt verwijderen?</target>
       </segment>
     </unit>
+    <unit id="RMPnq6o" name="label.username_or_email">
+      <segment>
+        <source>label.username_or_email</source>
+        <target>Gebruikersnaam of email</target>
```

Comment:
Dutch is stupid.

```suggestion
        <target>Gebruikersnaam of e-mail</target>
```

Reply:
Argh.. that reminds me of people in Bulgaria who refer to it not as "email" but "electronic post" :sweat_smile:   *cringe*